### PR TITLE
Add About page for Mobile App

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview

Fixes: [a random comment](https://zooniverse.slack.com/archives/C024GDV4Z/p1637929137053100) by @adammcmaster on Slack

This PR adds a simple About page for the Mobile App, since we don't have anything on our website to advertise our mobile stuff.

- Intended path is `https://www.zooniverse.org/about/mobile-app`
- ⚠️ Currently we have lorem ipsum text in place of anything less nonsensical. Copy editors required for this PR.
- ⚠️ An actual designer's eyes on this is also very very welcome

image

### Testing:

Test URL: https://pr-6062.pfe-preview.zooniverse.org/about/mobile-app
- This is a static content page
- Check that the outgoing URLs point to the Android app and iOS app respectively.
- Lorem Ipsum text shouldn't be there for the final review, obviously

### Status

WIP. We need actual text/copy (and some design touchups) to proceed.

⚠️ Also note to Shaun: don't forget to update the other language files.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
## PR Update

This PR is now ready for review! I've added the proper copy text, fixed a broken test, and the test URL remains: https://pr-6062.pfe-preview.zooniverse.org/about/mobile-app

Special 👍👍's to @trouille and @chelseatroy for authoring the text for this page!

**Comment by trouille** [comment]
The content in the preview link looks good. @mcbouslog Could you do a quick review from a dev perspective? And then it can merge! Thanks Shaun and Mark!

**Review by mcbouslog — APPROVED** [review]
LGTM!

2 super minor suggestions for consideration.

File: app/locales/en.js [hunk 1]
```diff
@@ -762,6 +763,21 @@ export default {
       toDownload: '[Download a pdf](%(url)s)',
       toPurchase: '[Purchase on %(purchaseSource)s](%(url)s)'
     },
+    mobileApp: {
+      title: '## Zooniverse Mobile App',
+      paragraphOne: 'The Zooniverse mobile app is a great way to get involved in Zooniverse projects on-the-go. Folks swipe on their phones to identify wildlife camera photos with animals in them. They draw rectangles around gravitational waves and tag distant galaxies. You can join the fun with your existing Zooniverse account, whether you have an iOS or an Android device!',
+      paragraphTwo: 'If you’re already working on a project that you think could go on mobile, please post in that project’s Talk discussion forum to start the conversation with the research team and us. The more, the merrier!',
+      paragraphThree: 'For more information about the Zooniverse mobile app, check out [https://blog.zooniverse.org/?s=mobile](https://blog.zooniverse.org/?s=mobile)',
```
Comment (mcbouslog) [inline app/locales/en.js:770]:
```suggestion
      paragraphThree: 'For more information about the Zooniverse mobile app, check out [https://blog.zooniverse.org/?s=mobile](https://blog.zooniverse.org/?s=mobile).',
```

File: app/locales/en.js [hunk 2]
```diff
@@ -762,6 +763,21 @@ export default {
...
+      paragraphFour: '**For researchers:** Mobile-enabled projects receive more classifications and so are able to finish more quickly. Please note: only certain kinds of projects can be enabled on mobile (see details within the blog posts linked above and instructions within the [Project Builder](http://zooniverse.org/lab) workflow editor interface). If you have any questions about the Zooniverse mobile app, email us at [contact@zooniverse.org!](mailto:contact@zooniverse.org)',
```
Comment (mcbouslog) [inline app/locales/en.js:771]:
```suggestion
      paragraphFour: '**For researchers:** Mobile-enabled projects receive more classifications and so are able to finish more quickly. Please note: only certain kinds of projects can be enabled on mobile (see details within the blog posts linked above and instructions within the [Project Builder](http://zooniverse.org/lab) workflow editor interface). If you have any questions about the Zooniverse mobile app, email us at [contact@zooniverse.org](mailto:contact@zooniverse.org)!',
```

**Comment by shaunanoordin** [comment]
Awesome, thanks Mark! I've committed your suggestions and will merge/deploy once tests are complete 👍

**Pull request merged**